src/OBD2/OBD2.c

```diff
@@ -24,6 +24,16 @@ int OBD2_get_current_PID_value(int index)
     return OBD2_current_values[index];
 }
 
+static float to_farenheight(float celcius)
+{
+    return celcius * 1.8 + 32.0;
+}
+
+static float to_mph(float kph)
```

Comment:
Ok. I'll update the names per suggested.  This will get worked over pretty hard in 2.9.0, though.